packages/core/src/components/toast/_toast.scss

```diff
@@ -103,8 +103,8 @@ $toast-intent-colors: (
   box-shadow: $pt-toast-box-shadow;
   display: flex;
   margin: $toast-margin 0 0;
-  max-width: $toast-max-width;
-  min-width: $toast-min-width;
+  max-width: min($toast-max-width, calc(100vw - $toast-margin));
+  min-width: max($toast-min-width, calc(100vw - $toast-margin));
```

Comment:
This sets `min-width` to be `100vw - $toast-margin` by default, which will end up stretching toasts across the screen.

![Screenshot 2024-09-11 at 10 26 56@2x](https://github.com/user-attachments/assets/b20454a8-62ab-4678-86f4-ae95874e1cb3)


I don't think this was the intended effect for the default experience. Would a `min()` or `clamp()` function work here instead?
```suggestion
  min-width: min($toast-min-width, calc(100vw - $toast-margin));
```

Reply:
gah yea I blindly made this change and hadn't tested it yet... thanks for actually testing (I'd like to think I would have before merging but 😅)

but yes this makes sense, we still want `min-width: $toast-min-width;`, unless the screen is smaller than that in which case we want `calc(100vw - $toast-margin)`